### ISSUE TITLE
Shift+Enter is ignored in comments

### ISSUE THREAD

**Issue by jzuken** [hwproj/hwproj#90]
acts like new line.

**Comment by Parean** [comment]
Does this happen only when user closes chat, and opens it again?

**Comment by jzuken** [comment]
nope, any time. just open the chat, enter some text, press shift-enter and it goes to a new line
